From a132a22e316121cf00ff733afb1ad1dc313e14b3 Mon Sep 17 00:00:00 2001
From: Jon Maloy <jmaloy@redhat.com>
Date: Wed, 30 Mar 2022 14:52:34 -0400
Subject: [PATCH 14/18] tests: acpi: whitelist nvdimm's SSDT and FACP.slic
 expected blobs

RH-Author: Jon Maloy <jmaloy@redhat.com>
RH-MergeRequest: 141: acpi: fix QEMU crash when started with SLIC table
RH-Commit: [6/10] 3f3a929cde82f228da1e4bc66e4c869467c0289c (jmaloy/qemu-kvm)
RH-Bugzilla: 2062611
RH-Acked-by: Igor Mammedov <imammedo@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=2062611
Upstream: Merged

commit d1e4a4654154925eddf0fc449fa9c92b806b9c8c
Author: Igor Mammedov <imammedo@redhat.com>
Date:   Wed Jan 12 08:03:30 2022 -0500

    tests: acpi: whitelist nvdimm's SSDT and FACP.slic expected blobs

    The next commit will revert OEM fields whitespace padding to
    padding with '\0' as it was before [1]. That will change OEM
    Table ID for:
      * SSDT.*: where it was padded from 6 characters to 8
      * FACP.slic: where it was padded from 2 characters to 8
    after reverting whitespace padding, it will be replaced with
    '\0' which effectively will shorten OEM table ID to 6 and 2
    characters.

    Whitelist affected tables before introducing the change.

    1) 602b458201 ("acpi: Permit OEM ID and OEM table ID fields to be changed")
    Signed-off-by: Igor Mammedov <imammedo@redhat.com>
    Message-Id: <20220112130332.1648664-3-imammedo@redhat.com>
    Reviewed-by: Michael S. Tsirkin <mst@redhat.com>
    Signed-off-by: Michael S. Tsirkin <mst@redhat.com>

(cherry picked from commit d1e4a4654154925eddf0fc449fa9c92b806b9c8c)
Signed-off-by: Jon Maloy <jmaloy@redhat.com>
---
 tests/qtest/bios-tables-test-allowed-diff.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/tests/qtest/bios-tables-test-allowed-diff.h b/tests/qtest/bios-tables-test-allowed-diff.h
index dfb8523c8b..7faa8f53be 100644
--- a/tests/qtest/bios-tables-test-allowed-diff.h
+++ b/tests/qtest/bios-tables-test-allowed-diff.h
@@ -1 +1,5 @@
 /* List of comma-separated changed AML files to ignore */
+"tests/data/acpi/virt/SSDT.memhp",
+"tests/data/acpi/pc/SSDT.dimmpxm",
+"tests/data/acpi/q35/SSDT.dimmpxm",
+"tests/data/acpi/q35/FACP.slic",
-- 
2.27.0

